src/components/HiddenTaskList/HiddenTaskList.jsx

```diff
@@ -70,14 +69,15 @@ class HiddenTaskList extends React.Component {
 
 HiddenTaskList.propTypes = {
   tasks: React.PropTypes.array.isRequired,
+  activeFilter: React.PropTypes.object.isRequired,
```

Comment:
We hit problems before where some components expected an object for the filter and others expected just the string name. Are you consistent throughout this branch with expecting the filter to be an object?